packages/widgets/Source/Viewer/Viewer.js

```diff
@@ -493,13 +473,16 @@ Either specify options.terrainProvider instead or set options.baseLayerPicker to
         ? false
         : undefined,
     clock: clock,
+    shouldAnimate: options.shouldAnimate,
```

Comment:
Can you please be sure to add these new `options` to the JSDoc for this class? The descriptions can match the ones from the `CesiumWidget` they're being passed to.

Reply:
`shouldAnimate` was already there in the `Viewer`'s ctor options before this PR. I added it to `CesiumWidget` ctor options to be passed to the `_clock`.
Did you mean something else?

Reply:
If they're all already there that's fine then. I was just looking at the git diff and didn't see them so probably missed it.